src/utils/utils_log.c

```diff
@@ -32,6 +32,29 @@
 #include "utils_common.h"
 #include "utils_log.h"
 
+#define UMF_VERSION_MAGIC_STR "\x00@(#) "
```

Comment:
out of curiosity - why this magic string? grepping with `UMF` should be fine as well, right?

Reply:
Grepping for "UMF" prints out a lot more strings (135 when I tried).